### PR TITLE
Implement Time property in WinUI TimePickerHandler

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Implement `Time` property in WinUI TimePickerHandler.

### PR Checklist ###

- [x] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)
- [x] Adds the property to the appropriate interface
- [x] Avoids any changes not essential to the handler property
- [x] Adds the mapping to the PropertyMapper in the handler
- [x] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [x] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [x] Tags ported renderer methods with [PortHandler]
- [x] Adds an example of the property to the sample project (MainPage)
- [x] Adds the property to the stub class
- [x] Implements basic property tests in DeviceTests

#### Does this PR touch anything that might effect accessibility?
No

### CODE REVIEW

**Review by rachelkang — COMMENTED** [review]
same comment as for #723 -
I can see the Time when the TimeePicker is initialized, but when I pick a different time, the TimePicker looks empty :/

File: src/Core/src/Platform/Windows/TimePickerExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace Microsoft.Maui
+{
+	public static class TimePickerExtensions
+	{
+		public static void UpdateTime(this TimePicker nativeTimePicker, ITimePicker timePicker)
+		{
+			nativeTimePicker.Time = timePicker.Time;
+
+			if (timePicker.Format?.Contains('H') == true)
```
Comment (rmarinho) [inline src/Core/src/Platform/Windows/TimePickerExtensions.cs:11]: Should this be it's own mapper?

**Comment by rmarinho** [comment]
Seems to be working, just a question about format should use the mapper or not .. 

image
image
image
image

**Review by rmarinho — APPROVED** [review]

**Comment by rachelkang** [comment]
trying again now

**Review by rachelkang — APPROVED** [review]
LGTM

**Pull request merged**